src/valkey-benchmark.c

```diff
@@ -2338,6 +2338,12 @@ int main(int argc, char **argv) {
             free(cmd);
         }
 
+        if (test_is_selected("sismember")) {
```

Comment:
Let's move this to before the spop test above. Otherwise, I guess the spop test deletes all the elements in the set.